### DISCUSSION TITLE
Is there a Mudry Cap10B aircraft in JSBSim?

### DISCUSSION THREAD

**Discussion by theorynice** [JSBSim-Team/jsbsim#671] [General]
Excuse me, is there a Mudry Cap10B aircraft in JSBSim? I try to check in the aircraft folder but find no related data. I am grateful if anyone can supply it.

**Comment by bcoconni** [comment]
There is a [JSBSim model for the Cap10B in FlightGear hangar](https://sourceforge.net/p/flightgear/fgaddon/HEAD/tree/trunk/Aircraft/Cap10B/). Be aware however that some JSBSim flight models from FlightGear need some cleanup to be used in JSBSim standalone.

**Reply by bcoconni** [reply]
While skimming through the model I saw that @dany93 is one of the author of this FDM so he might be able to answer your questions, if you have any.

**Reply by theorynice** [reply]
Thanks very much!

**Comment by dany93** [comment]
The initial source is Patten Aircraft Hangar http://pattenflightgear.wifeo.com/avions.php
Among other ones,  you can download the Cap10B from here http://pattenflightgear.wifeo.com/cap-10-b.php. Maybe slightly more up to date.

**Reply by theorynice** [reply]
Thanks very much!
